### DISCUSSION TITLE
Can't I use toggleSidebar like v1?

### DISCUSSION THREAD

**Discussion by giwonn** [vuepress/core#575] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/72638829/146681475-ef56dcf3-fbb8-4aa4-b552-32572a9dccd1.png)

This image is sidebar of my blog using V1.

I want to use toggleSidebar.

But I didn't find how to use toggleSidebar.

Has this function not been implemented yet?

I saw documentation about toggleSidebar
[https://v2.vuepress.vuejs.org/reference/default-theme/config.html#togglesidebar](https://v2.vuepress.vuejs.org/reference/default-theme/config.html#togglesidebar)

But that is not what I want.

**Comment by Mister-Hope** [comment, marked as answer]
What do you mean by toggle sidebar?

V2 has implemented most of the features in v1, so you'd better describe what you want.

Also, you can always choose to extend `@vuepress/theme-default` to achive project you like.

**Reply by giwonn** [reply]
I already fixed this issue. Thank you :)
